raft/server.cc

```diff
@@ -13,6 +13,7 @@
 #include <boost/range/adaptor/map.hpp>
 #include <boost/range/algorithm/copy.hpp>
 #include <boost/range/join.hpp>
+#include <boost/lexical_cast.hpp>
```

Comment:
Why not convert the error injection point in override_snapshot_thresholds too?

Reply:
Because I want to keep the patch localized and not grow it.